packages/pulseaudio/build.sh

```diff
@@ -4,7 +4,8 @@ TERMUX_PKG_LICENSE="GPL-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_SRCURL=https://github.com/pulseaudio/pulseaudio.git
 TERMUX_PKG_VERSION=14.2
-TERMUX_PKG_DEPENDS="libltdl, libsndfile, libandroid-glob, libsoxr, speexdsp"
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_DEPENDS="libltdl, libsndfile, libandroid-glob, libsoxr, speexdsp, libwebrtc-audio-processsing"
```

Comment:
There's a typo in `libwebrtc-audio-processsing` (should be only two 's')

Reply:
Oops, looks like I made a typo while resolving merge conflicts from my phone. Thanks for catching that.